Running ./build.cmd doesn't work

I just cloned and then immediately tried to run ./build.cmd. Hopefully the error message helps you.

```
PS C:\Users\ethanseal\Documents\msbuild> .\build.cmd
  Determining projects to restore...
  All projects are up-to-date for restore.
  Determining projects to restore...
  All projects are up-to-date for restore.
  Microsoft.Build.Framework -> C:\Users\ethanseal\Documents\msbuild\artifacts\bin\Microsoft.Build.Framework\Debug\net47
  2\Microsoft.Build.Framework.dll
  Dependency -> C:\Users\ethanseal\Documents\msbuild\artifacts\bin\Samples\Dependency\Debug\net472\Dependency.dll
  TaskUsageLogger -> C:\Users\ethanseal\Documents\msbuild\artifacts\bin\Samples\TaskUsageLogger\Debug\net472\TaskUsageL
  ogger.dll
  TaskUsageLogger -> C:\Users\ethanseal\Documents\msbuild\artifacts\bin\Samples\TaskUsageLogger\Debug\netstandard2.0\Ta
  skUsageLogger.dll
  Dependency -> C:\Users\ethanseal\Documents\msbuild\artifacts\bin\Samples\Dependency\Debug\netstandard2.0\Dependency.d
  ll
  TaskUsageLogger -> C:\Users\ethanseal\Documents\msbuild\artifacts\bin\Samples\TaskUsageLogger\Debug\net7.0\TaskUsageL
  ogger.dll
  Dependency -> C:\Users\ethanseal\Documents\msbuild\artifacts\bin\Samples\Dependency\Debug\net7.0\Dependency.dll
  PortableTask -> C:\Users\ethanseal\Documents\msbuild\artifacts\bin\Samples\PortableTask\Debug\netstandard2.0\Portable
  Task.dll
  XmlFileLogger -> C:\Users\ethanseal\Documents\msbuild\artifacts\bin\Samples\XmlFileLogger\Debug\net472\XmlFileLogger.
  dll
  XmlFileLogger -> C:\Users\ethanseal\Documents\msbuild\artifacts\bin\Samples\XmlFileLogger\Debug\netstandard2.0\XmlFil
  eLogger.dll
  Xunit.NetCore.Extensions -> C:\Users\ethanseal\Documents\msbuild\artifacts\bin\Xunit.NetCore.Extensions\Debug\netstan
  dard2.0\Xunit.NetCore.Extensions.dll
  Xunit.NetCore.Extensions -> C:\Users\ethanseal\Documents\msbuild\artifacts\bin\Xunit.NetCore.Extensions\Debug\net472\
  Xunit.NetCore.Extensions.dll
  Xunit.NetCore.Extensions -> C:\Users\ethanseal\Documents\msbuild\artifacts\bin\Xunit.NetCore.Extensions\Debug\net7.0\
  Xunit.NetCore.Extensions.dll
  Microsoft.Build.UnGAC -> C:\Users\ethanseal\Documents\msbuild\artifacts\bin\Microsoft.Build.UnGAC\Debug\net45\Microso
  ft.Build.UnGAC.exe
  StringTools -> C:\Users\ethanseal\Documents\msbuild\artifacts\bin\StringTools\Debug\net35\Microsoft.NET.StringTools.n
  et35.dll
  XmlFileLogger -> C:\Users\ethanseal\Documents\msbuild\artifacts\bin\Samples\XmlFileLogger\Debug\net7.0\XmlFileLogger.
  dll
  StringTools -> C:\Users\ethanseal\Documents\msbuild\artifacts\bin\StringTools\Debug\net472\Microsoft.NET.StringTools.
  dll
  MSBuildTaskHost -> C:\Users\ethanseal\Documents\msbuild\artifacts\bin\MSBuildTaskHost\Debug\net35\MSBuildTaskHost.exe
  StringTools -> C:\Users\ethanseal\Documents\msbuild\artifacts\bin\StringTools\Debug\netstandard2.0\Microsoft.NET.Stri
  ngTools.dll
  StringTools.UnitTests.net35 -> C:\Users\ethanseal\Documents\msbuild\artifacts\bin\StringTools.UnitTests.net35\Debug\n
  et472\Microsoft.NET.StringTools.net35.UnitTests.dll
  Microsoft.Build.Framework -> C:\Users\ethanseal\Documents\msbuild\artifacts\bin\Microsoft.Build.Framework\Debug\net7.
  0\Microsoft.Build.Framework.dll
  StringTools.UnitTests -> C:\Users\ethanseal\Documents\msbuild\artifacts\bin\StringTools.UnitTests\Debug\net472\Micros
  oft.NET.StringTools.UnitTests.dll
  StringTools -> C:\Users\ethanseal\Documents\msbuild\artifacts\bin\StringTools\Debug\net7.0\Microsoft.NET.StringTools.
  dll
  Microsoft.Build.Framework -> C:\Users\ethanseal\Documents\msbuild\artifacts\bin\Microsoft.Build.Framework\Debug\netst
  andard2.0\Microsoft.Build.Framework.dll
  Microsoft.Build.Framework.UnitTests -> C:\Users\ethanseal\Documents\msbuild\artifacts\bin\Microsoft.Build.Framework.U
  nitTests\Debug\net472\Microsoft.Build.Framework.UnitTests.dll
  StringTools.Benchmark -> C:\Users\ethanseal\Documents\msbuild\artifacts\bin\StringTools.Benchmark\Debug\net472\String
  Tools.Benchmark.exe
  Microsoft.Build.Utilities -> C:\Users\ethanseal\Documents\msbuild\artifacts\bin\Microsoft.Build.Utilities\Debug\net7.
  0\Microsoft.Build.Utilities.Core.dll
  Microsoft.Build.Utilities -> C:\Users\ethanseal\Documents\msbuild\artifacts\bin\Microsoft.Build.Utilities\Debug\net47
  2\Microsoft.Build.Utilities.Core.dll
  Microsoft.Build.Engine -> C:\Users\ethanseal\Documents\msbuild\artifacts\bin\Microsoft.Build.Engine\Debug\net472\Micr
  osoft.Build.Engine.dll
  StringTools.Benchmark -> C:\Users\ethanseal\Documents\msbuild\artifacts\bin\StringTools.Benchmark\Debug\net7.0\String
  Tools.Benchmark.dll
  Microsoft.Build -> C:\Users\ethanseal\Documents\msbuild\artifacts\bin\Microsoft.Build\Debug\net7.0\Microsoft.Build.dl
  l
  Microsoft.Build -> C:\Users\ethanseal\Documents\msbuild\artifacts\bin\Microsoft.Build\Debug\net472\Microsoft.Build.dl
  l
  Microsoft.Build.Utilities -> C:\Users\ethanseal\Documents\msbuild\artifacts\bin\Microsoft.Build.Utilities\Debug\netst
  andard2.0\Microsoft.Build.Utilities.Core.dll
  TaskWithDependency -> C:\Users\ethanseal\Documents\msbuild\artifacts\bin\Samples\TaskWithDependency\Debug\net7.0\Task
  WithDependency.dll
  StringTools.UnitTests -> C:\Users\ethanseal\Documents\msbuild\artifacts\bin\StringTools.UnitTests\Debug\net7.0\Micros
  oft.NET.StringTools.UnitTests.dll
  Microsoft.Build.Framework.UnitTests -> C:\Users\ethanseal\Documents\msbuild\artifacts\bin\Microsoft.Build.Framework.U
  nitTests\Debug\net7.0\Microsoft.Build.Framework.UnitTests.dll
  Microsoft.Build.Tasks -> C:\Users\ethanseal\Documents\msbuild\artifacts\bin\Microsoft.Build.Tasks\Debug\net7.0\Micros
  oft.Build.Tasks.Core.dll
  Microsoft.Build.Tasks -> C:\Users\ethanseal\Documents\msbuild\artifacts\bin\Microsoft.Build.Tasks\Debug\netstandard2.
  0\Microsoft.Build.Tasks.Core.dll
  ProjectCachePlugin -> C:\Users\ethanseal\Documents\msbuild\artifacts\bin\Samples\ProjectCachePlugin\Debug\net7.0\Proj
  ectCachePlugin.dll
  ProjectCachePlugin -> C:\Users\ethanseal\Documents\msbuild\artifacts\bin\Samples\ProjectCachePlugin\Debug\net472\Proj
  ectCachePlugin.dll
  TaskWithDependency -> C:\Users\ethanseal\Documents\msbuild\artifacts\bin\Samples\TaskWithDependency\Debug\netstandard
  2.0\TaskWithDependency.dll
  TaskWithDependency -> C:\Users\ethanseal\Documents\msbuild\artifacts\bin\Samples\TaskWithDependency\Debug\net472\Task
  WithDependency.dll
  Microsoft.Build.Conversion -> C:\Users\ethanseal\Documents\msbuild\artifacts\bin\Microsoft.Build.Conversion\Debug\net
  472\Microsoft.Build.Conversion.Core.dll
  Microsoft.Build.Tasks -> C:\Users\ethanseal\Documents\msbuild\artifacts\bin\Microsoft.Build.Tasks\Debug\net472\Micros
  oft.Build.Tasks.Core.dll
  Microsoft.Build.Engine.OM.UnitTests -> C:\Users\ethanseal\Documents\msbuild\artifacts\bin\Microsoft.Build.Engine.OM.U
  nitTests\Debug\net7.0\Microsoft.Build.Engine.OM.UnitTests.dll
  MSBuild.VSSetup.Arm64 -> C:\Users\ethanseal\Documents\msbuild\artifacts\bin\MSBuild.VSSetup.Arm64\Debug\net472\MSBuil
  d.VSSetup.Arm64.dll
  MSBuild -> C:\Users\ethanseal\Documents\msbuild\artifacts\bin\MSBuild\Debug\net7.0\MSBuild.dll
  MSBuild -> C:\Users\ethanseal\Documents\msbuild\artifacts\bin\MSBuild\Debug\net472\MSBuild.exe
  Microsoft.Build.Engine.OM.UnitTests -> C:\Users\ethanseal\Documents\msbuild\artifacts\bin\Microsoft.Build.Engine.OM.U
  nitTests\Debug\net472\Microsoft.Build.Engine.OM.UnitTests.dll
  MSBuild -> C:\Users\ethanseal\Documents\msbuild\artifacts\bin\MSBuild\arm64\Debug\net472\MSBuild.exe
  Localization -> C:\Users\ethanseal\Documents\msbuild\artifacts\bin\Localization\Debug\net7.0\Localization.dll
  MSBuild.Bootstrap -> C:\Users\ethanseal\Documents\msbuild\artifacts\bin\MSBuild.Bootstrap\Debug\net7.0\MSBuild.Bootst
  rap.dll
  Microsoft.Build.Utilities.UnitTests -> C:\Users\ethanseal\Documents\msbuild\artifacts\bin\Microsoft.Build.Utilities.U
  nitTests\Debug\net7.0\Microsoft.Build.Utilities.UnitTests.dll
  Microsoft.Build.CommandLine.UnitTests -> C:\Users\ethanseal\Documents\msbuild\artifacts\bin\Microsoft.Build.CommandLi
  ne.UnitTests\Debug\net7.0\Microsoft.Build.CommandLine.UnitTests.dll
  Microsoft.Build.CommandLine.UnitTests -> C:\Users\ethanseal\Documents\msbuild\artifacts\bin\Microsoft.Build.CommandLi
  ne.UnitTests\Debug\net472\Microsoft.Build.CommandLine.UnitTests.dll
  Microsoft.Build.Utilities.UnitTests -> C:\Users\ethanseal\Documents\msbuild\artifacts\bin\Microsoft.Build.Utilities.U
  nitTests\Debug\net472\Microsoft.Build.Utilities.UnitTests.dll
  Microsoft.Build.Tasks.UnitTests -> C:\Users\ethanseal\Documents\msbuild\artifacts\bin\Microsoft.Build.Tasks.UnitTests
  \Debug\net7.0\Microsoft.Build.Tasks.UnitTests.dll
  Microsoft.Build.Tasks.UnitTests -> C:\Users\ethanseal\Documents\msbuild\artifacts\bin\Microsoft.Build.Tasks.UnitTests
  \Debug\net472\Microsoft.Build.Tasks.UnitTests.dll
  Microsoft.Build.Engine.UnitTests -> C:\Users\ethanseal\Documents\msbuild\artifacts\bin\Microsoft.Build.Engine.UnitTes
  ts\Debug\net7.0\Microsoft.Build.Engine.UnitTests.dll
  DevDivPackage -> C:\Users\ethanseal\Documents\msbuild\artifacts\bin\DevDivPackage\Debug\net472\DevDivPackage.dll
  Microsoft.Build.Engine.UnitTests -> C:\Users\ethanseal\Documents\msbuild\artifacts\bin\Microsoft.Build.Engine.UnitTes
  ts\Debug\net472\Microsoft.Build.Engine.UnitTests.dll
  MSBuild.Bootstrap -> C:\Users\ethanseal\Documents\msbuild\artifacts\bin\MSBuild.Bootstrap\Debug\net7.0\net7.0\publish
  \
  MSBuild.VSSetup -> C:\Users\ethanseal\Documents\msbuild\artifacts\bin\MSBuild.VSSetup\Debug\net472\MSBuild.VSSetup.dl
  l
  MSBuild -> C:\Users\ethanseal\Documents\msbuild\artifacts\bin\MSBuild\x64\Debug\net472\MSBuild.exe
  MSBuild.Bootstrap -> C:\Users\ethanseal\Documents\msbuild\artifacts\bin\MSBuild.Bootstrap\Debug\net472\MSBuild.Bootst
  rap.dll
  MSBuildTaskHost -> C:\Users\ethanseal\Documents\msbuild\artifacts\bin\MSBuildTaskHost\x64\Debug\net35\MSBuildTaskHost
  .exe
C:\Users\ethanseal\.nuget\packages\microsoft.dotnet.arcade.sdk\6.0.0-beta.22458.2\tools\VisualStudio.BuildIbcTrainingIn
puts.targets(21,5): error MSB4018: The "GenerateTrainingInputFiles" task failed unexpectedly. [C:\Users\ethanseal\.nuge
t\packages\microsoft.dotnet.arcade.sdk\6.0.0-beta.22458.2\tools\AfterSigning.proj]
C:\Users\ethanseal\.nuget\packages\microsoft.dotnet.arcade.sdk\6.0.0-beta.22458.2\tools\VisualStudio.BuildIbcTrainingIn
puts.targets(21,5): error MSB4018: System.IO.DirectoryNotFoundException: Could not find a part of the path 'C:\Users\et
hanseal\Documents\msbuild\artifacts\OptProf\Debug\Data\Microsoft.VisualStudio.ProjectSystem.DDRIT\Configurations\Micros
oft.VisualStudio.ProjectSystem.DDRIT.OptProfOpenCloseTest.OpenAndCloseProjectTestSolution\Microsoft.Build.Conversion.Co
re.0.IBC.json'. [C:\Users\ethanseal\.nuget\packages\microsoft.dotnet.arcade.sdk\6.0.0-beta.22458.2\tools\AfterSigning.p
roj]
C:\Users\ethanseal\.nuget\packages\microsoft.dotnet.arcade.sdk\6.0.0-beta.22458.2\tools\VisualStudio.BuildIbcTrainingIn
puts.targets(21,5): error MSB4018:    at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath) [C:\Users\
ethanseal\.nuget\packages\microsoft.dotnet.arcade.sdk\6.0.0-beta.22458.2\tools\AfterSigning.proj]
C:\Users\ethanseal\.nuget\packages\microsoft.dotnet.arcade.sdk\6.0.0-beta.22458.2\tools\VisualStudio.BuildIbcTrainingIn
puts.targets(21,5): error MSB4018:    at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32
 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, Strin
g msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost) [C:\Users\ethanseal\.nuget\packages\microsoft.do
tnet.arcade.sdk\6.0.0-beta.22458.2\tools\AfterSigning.proj]
C:\Users\ethanseal\.nuget\packages\microsoft.dotnet.arcade.sdk\6.0.0-beta.22458.2\tools\VisualStudio.BuildIbcTrainingIn
puts.targets(21,5): error MSB4018:    at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, File
Share share) [C:\Users\ethanseal\.nuget\packages\microsoft.dotnet.arcade.sdk\6.0.0-beta.22458.2\tools\AfterSigning.proj
]
C:\Users\ethanseal\.nuget\packages\microsoft.dotnet.arcade.sdk\6.0.0-beta.22458.2\tools\VisualStudio.BuildIbcTrainingIn
puts.targets(21,5): error MSB4018:    at Microsoft.DotNet.Build.Tasks.VisualStudio.GenerateTrainingInputFiles.WriteEntr
ies(IbcEntry[] ibcEntries, String outDir) in /_/src/Microsoft.DotNet.Build.Tasks.VisualStudio/OptProf/GenerateTrainingI
nputFiles.cs:line 157 [C:\Users\ethanseal\.nuget\packages\microsoft.dotnet.arcade.sdk\6.0.0-beta.22458.2\tools\AfterSig
ning.proj]
C:\Users\ethanseal\.nuget\packages\microsoft.dotnet.arcade.sdk\6.0.0-beta.22458.2\tools\VisualStudio.BuildIbcTrainingIn
puts.targets(21,5): error MSB4018:    at Microsoft.DotNet.Build.Tasks.VisualStudio.GenerateTrainingInputFiles.WriteEntr
ies(OptProfTrainingTest[] tests, IbcEntry[] ibcEntries) in /_/src/Microsoft.DotNet.Build.Tasks.VisualStudio/OptProf/Gen
erateTrainingInputFiles.cs:line 123 [C:\Users\ethanseal\.nuget\packages\microsoft.dotnet.arcade.sdk\6.0.0-beta.22458.2\
tools\AfterSigning.proj]
C:\Users\ethanseal\.nuget\packages\microsoft.dotnet.arcade.sdk\6.0.0-beta.22458.2\tools\VisualStudio.BuildIbcTrainingIn
puts.targets(21,5): error MSB4018:    at Microsoft.DotNet.Build.Tasks.VisualStudio.GenerateTrainingInputFiles.ExecuteIm
pl() in /_/src/Microsoft.DotNet.Build.Tasks.VisualStudio/OptProf/GenerateTrainingInputFiles.cs:line 83 [C:\Users\ethans
eal\.nuget\packages\microsoft.dotnet.arcade.sdk\6.0.0-beta.22458.2\tools\AfterSigning.proj]
C:\Users\ethanseal\.nuget\packages\microsoft.dotnet.arcade.sdk\6.0.0-beta.22458.2\tools\VisualStudio.BuildIbcTrainingIn
puts.targets(21,5): error MSB4018:    at Microsoft.DotNet.Build.Tasks.VisualStudio.GenerateTrainingInputFiles.Execute()
 in /_/src/Microsoft.DotNet.Build.Tasks.VisualStudio/OptProf/GenerateTrainingInputFiles.cs:line 41 [C:\Users\ethanseal\
.nuget\packages\microsoft.dotnet.arcade.sdk\6.0.0-beta.22458.2\tools\AfterSigning.proj]
C:\Users\ethanseal\.nuget\packages\microsoft.dotnet.arcade.sdk\6.0.0-beta.22458.2\tools\VisualStudio.BuildIbcTrainingIn
puts.targets(21,5): error MSB4018:    at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecuti
onHost.Execute() [C:\Users\ethanseal\.nuget\packages\microsoft.dotnet.arcade.sdk\6.0.0-beta.22458.2\tools\AfterSigning.
proj]
C:\Users\ethanseal\.nuget\packages\microsoft.dotnet.arcade.sdk\6.0.0-beta.22458.2\tools\VisualStudio.BuildIbcTrainingIn
puts.targets(21,5): error MSB4018:    at Microsoft.Build.BackEnd.TaskBuilder.<ExecuteInstantiatedTask>d__26.MoveNext()
[C:\Users\ethanseal\.nuget\packages\microsoft.dotnet.arcade.sdk\6.0.0-beta.22458.2\tools\AfterSigning.proj]

Build FAILED.

C:\Users\ethanseal\.nuget\packages\microsoft.dotnet.arcade.sdk\6.0.0-beta.22458.2\tools\VisualStudio.BuildIbcTrainingIn
puts.targets(21,5): error MSB4018: The "GenerateTrainingInputFiles" task failed unexpectedly. [C:\Users\ethanseal\.nuge
t\packages\microsoft.dotnet.arcade.sdk\6.0.0-beta.22458.2\tools\AfterSigning.proj]
C:\Users\ethanseal\.nuget\packages\microsoft.dotnet.arcade.sdk\6.0.0-beta.22458.2\tools\VisualStudio.BuildIbcTrainingIn
puts.targets(21,5): error MSB4018: System.IO.DirectoryNotFoundException: Could not find a part of the path 'C:\Users\et
hanseal\Documents\msbuild\artifacts\OptProf\Debug\Data\Microsoft.VisualStudio.ProjectSystem.DDRIT\Configurations\Micros
oft.VisualStudio.ProjectSystem.DDRIT.OptProfOpenCloseTest.OpenAndCloseProjectTestSolution\Microsoft.Build.Conversion.Co
re.0.IBC.json'. [C:\Users\ethanseal\.nuget\packages\microsoft.dotnet.arcade.sdk\6.0.0-beta.22458.2\tools\AfterSigning.p
roj]
C:\Users\ethanseal\.nuget\packages\microsoft.dotnet.arcade.sdk\6.0.0-beta.22458.2\tools\VisualStudio.BuildIbcTrainingIn
puts.targets(21,5): error MSB4018:    at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath) [C:\Users\
ethanseal\.nuget\packages\microsoft.dotnet.arcade.sdk\6.0.0-beta.22458.2\tools\AfterSigning.proj]
C:\Users\ethanseal\.nuget\packages\microsoft.dotnet.arcade.sdk\6.0.0-beta.22458.2\tools\VisualStudio.BuildIbcTrainingIn
puts.targets(21,5): error MSB4018:    at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32
 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, Strin
g msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost) [C:\Users\ethanseal\.nuget\packages\microsoft.do
tnet.arcade.sdk\6.0.0-beta.22458.2\tools\AfterSigning.proj]
C:\Users\ethanseal\.nuget\packages\microsoft.dotnet.arcade.sdk\6.0.0-beta.22458.2\tools\VisualStudio.BuildIbcTrainingIn
puts.targets(21,5): error MSB4018:    at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, File
Share share) [C:\Users\ethanseal\.nuget\packages\microsoft.dotnet.arcade.sdk\6.0.0-beta.22458.2\tools\AfterSigning.proj
]
C:\Users\ethanseal\.nuget\packages\microsoft.dotnet.arcade.sdk\6.0.0-beta.22458.2\tools\VisualStudio.BuildIbcTrainingIn
puts.targets(21,5): error MSB4018:    at Microsoft.DotNet.Build.Tasks.VisualStudio.GenerateTrainingInputFiles.WriteEntr
ies(IbcEntry[] ibcEntries, String outDir) in /_/src/Microsoft.DotNet.Build.Tasks.VisualStudio/OptProf/GenerateTrainingI
nputFiles.cs:line 157 [C:\Users\ethanseal\.nuget\packages\microsoft.dotnet.arcade.sdk\6.0.0-beta.22458.2\tools\AfterSig
ning.proj]
C:\Users\ethanseal\.nuget\packages\microsoft.dotnet.arcade.sdk\6.0.0-beta.22458.2\tools\VisualStudio.BuildIbcTrainingIn
puts.targets(21,5): error MSB4018:    at Microsoft.DotNet.Build.Tasks.VisualStudio.GenerateTrainingInputFiles.WriteEntr
ies(OptProfTrainingTest[] tests, IbcEntry[] ibcEntries) in /_/src/Microsoft.DotNet.Build.Tasks.VisualStudio/OptProf/Gen
erateTrainingInputFiles.cs:line 123 [C:\Users\ethanseal\.nuget\packages\microsoft.dotnet.arcade.sdk\6.0.0-beta.22458.2\
tools\AfterSigning.proj]
C:\Users\ethanseal\.nuget\packages\microsoft.dotnet.arcade.sdk\6.0.0-beta.22458.2\tools\VisualStudio.BuildIbcTrainingIn
puts.targets(21,5): error MSB4018:    at Microsoft.DotNet.Build.Tasks.VisualStudio.GenerateTrainingInputFiles.ExecuteIm
pl() in /_/src/Microsoft.DotNet.Build.Tasks.VisualStudio/OptProf/GenerateTrainingInputFiles.cs:line 83 [C:\Users\ethans
eal\.nuget\packages\microsoft.dotnet.arcade.sdk\6.0.0-beta.22458.2\tools\AfterSigning.proj]
C:\Users\ethanseal\.nuget\packages\microsoft.dotnet.arcade.sdk\6.0.0-beta.22458.2\tools\VisualStudio.BuildIbcTrainingIn
puts.targets(21,5): error MSB4018:    at Microsoft.DotNet.Build.Tasks.VisualStudio.GenerateTrainingInputFiles.Execute()
 in /_/src/Microsoft.DotNet.Build.Tasks.VisualStudio/OptProf/GenerateTrainingInputFiles.cs:line 41 [C:\Users\ethanseal\
.nuget\packages\microsoft.dotnet.arcade.sdk\6.0.0-beta.22458.2\tools\AfterSigning.proj]
C:\Users\ethanseal\.nuget\packages\microsoft.dotnet.arcade.sdk\6.0.0-beta.22458.2\tools\VisualStudio.BuildIbcTrainingIn
puts.targets(21,5): error MSB4018:    at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecuti
onHost.Execute() [C:\Users\ethanseal\.nuget\packages\microsoft.dotnet.arcade.sdk\6.0.0-beta.22458.2\tools\AfterSigning.
proj]
C:\Users\ethanseal\.nuget\packages\microsoft.dotnet.arcade.sdk\6.0.0-beta.22458.2\tools\VisualStudio.BuildIbcTrainingIn
puts.targets(21,5): error MSB4018:    at Microsoft.Build.BackEnd.TaskBuilder.<ExecuteInstantiatedTask>d__26.MoveNext()
[C:\Users\ethanseal\.nuget\packages\microsoft.dotnet.arcade.sdk\6.0.0-beta.22458.2\tools\AfterSigning.proj]
    0 Warning(s)
    1 Error(s)

Time Elapsed 00:00:12.56
Build failed with exit code 1. Check errors above.
PS C:\Users\ethanseal\Documents\msbuild>
```

### Issue Description
<!--
* Please include a clear and concise description of the problem.
-->

### Steps to Reproduce
<!--
* Include as much of the following as possible:

* A minimal sample project that reproduces the issue.
* Your zipped project.
* IDE / CLI steps to create the project and reproduce the behaviour.
* Your command line invocation

If providing steps to reproduce the issue proves difficult, consider attaching a binlog:
[Details on sharing binary logs](https://aka.ms/msbuild/binlog)
[More information on binary logs](https://github.com/dotnet/msbuild/blob/main/documentation/wiki/Binary-Log.md)
NOTE: Binary logs capture environment information. Ensure it does not include any information you don't want to be public before you attach one.
If you want to share one just with Microsoft, you can [report a problem through Visual Studio](https://developercommunity.visualstudio.com/report) and share it privately with Microsoft.
-->

### Expected Behavior
<!--
* The expected output or behavior.
-->

### Actual Behavior
<!--
* The actual output or behavior.
-->

### Analysis
<!--
* If you have an idea where the problem might lie, let us know that here.
* Please include any pointers to code, relevant changes, or related issues you know of.
-->

### Versions & Configurations
<!--
* In a Visual Studio developer command prompt, run `msbuild -version` and paste the output here.
* If applicable, include the version of the tool that invokes MSBuild (Visual Studio, dotnet CLI, etc):

* Post any other relevant configuration settings here.
*   OS, architecture, etc.
-->

--- a/README.md
+++ b/README.md
@@ -41,11 +41,12 @@ To get started on **Visual Studio 2022**:
 1. [Install Visual Studio 2022](https://www.visualstudio.com/vs/).  Select the following Workloads:
   - .NET desktop development
   - .NET Core cross-platform development
-2. Open a `Developer Command Prompt for VS 2022` prompt.
-3. Clone the source code: `git clone https://github.com/dotnet/msbuild`
+2. Ensure [long path support](https://learn.microsoft.com/windows/win32/fileio/maximum-file-path-limitation?tabs=registry#enable-long-paths-in-windows-10-version-1607-and-later) is enabled at the Windows level.
+3. Open a `Developer Command Prompt for VS 2022` prompt.
+4. Clone the source code: `git clone https://github.com/dotnet/msbuild`
   - You may have to [download Git](https://git-scm.com/downloads) first.
-4. Run `.\build.cmd` from the root of the repo to build the code. This also restores packages needed to open the projects in Visual Studio.
-5. Open `MSBuild.sln` or `MSBuild.Dev.slnf` in Visual Studio 2022.
+5. Run `.\build.cmd` from the root of the repo to build the code. This also restores packages needed to open the projects in Visual Studio.
+6. Open `MSBuild.sln` or `MSBuild.Dev.slnf` in Visual Studio 2022.
 
 Note: To create a usable MSBuild with your changes, run `.\build.cmd /p:CreateBootstrap=true`.
 To build release, add `-c Release`: `.\build.cmd -c Release /p:CreateBootstrap=true`.

